pallets/messages/src/benchmarking.rs

```diff
@@ -35,6 +35,20 @@ benchmarks! {
 		}
 	}: _ (RawOrigin::Signed(caller), None, 1, input)
 
+	add_ipfs_message {
+		let n in 0 .. T::MaxMessagePayloadSizeBytes::get() - 1;
+		let m in 1 .. MESSAGES;
+		let caller: T::AccountId = whitelisted_caller();
+		let input = vec![1; n as usize];
+		let cid = CIDv2::new(input);
+		let payload_length = 1_000;
+
+		// for j in 0 .. m {
+		// 	let sid = j % SCHEMAS;
+		// 	assert_ok!(add_message::<T>(sid.try_into().unwrap()));
+		// }
+	}: _ (RawOrigin::Signed(caller), None, 65535, cid, payload_length)
```

Comment:
So when you are using `..` in benchmarks that means it should be used in some shape or form as an input. Instead of payload_length you probably should pass n inside here.

random numbers like `65535` are confusing

Reply:
Also why does this use CIDv2? That's not real.